README.md

```diff
@@ -1 +1,40 @@
-# code-review
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Peer to peer code review
```

Comment:
Hey @ksigei !
You have done a good job adding the project name !  in the README file 👏 
But there are some staffs that are not done yet !

- Please, add a screenshot to your README file to make it more clear and professional !